### ISSUE TITLE
need parser to support PAGE_CHECKSUM and TRANSACTIONAL

### ISSUE THREAD

**Issue by at009** [pingcap/tidb#45223]
## Feature Request

is it possible to update parser to support PAGE_CHECKSUM=1 TRANSACTIONAL=1 in create table statement ?

CREATE TABLE t (
  col1 int(10) unsigned NOT NULL,
  col2 int(10) unsigned NOT NULL,
  col3 int(11) NOT NULL DEFAULT 0,
  col4 tinyint(3) unsigned NOT NULL DEFAULT 0,
 col5 char(8) NOT NULL DEFAULT '',
  PRIMARY KEY (col1,col2)
) ENGINE=Aria  PAGE_CHECKSUM=1 TRANSACTIONAL=1

Getting error 'error: line 1 column 71 near "PAGE_CHECKSUM=1 TRANSACTIONAL=1"' while parsing the above DDL. This issue started after mariadb upgrade from 10.3 to 10.6.

**Comment by Defined2014** [comment]
I think it is not a mysql standard.

```
mysql> CREATE TABLE t (
    -> col1 int(10) unsigned NOT NULL,
    -> col2 int(10) unsigned NOT NULL,
    -> col3 int(11) NOT NULL DEFAULT 0,
    -> col4 tinyint(3) unsigned NOT NULL DEFAULT 0,
    -> col5 char(8) NOT NULL DEFAULT '',
    -> PRIMARY KEY (col1,col2)
    -> ) ENGINE=Aria PAGE_CHECKSUM=1 TRANSACTIONAL=1;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PAGE_CHECKSUM=1 TRANSACTIONAL=1' at line 8
```

**Comment by at009** [comment]
yes, we are using mariadb and we are looking for parser to support PAGE_CHECKSUM=1 TRANSACTIONAL=1.
The issue is only with mariadb 10.6 .
Is there any suggestion to fix this issue ?
